server/validators/employer.ts

```diff
@@ -153,6 +153,7 @@ export const EmployerFormSchema = yup
       .test('is-true', errorMessage, (v) => v === true),
   });
 
+// TODO: determine if this is needed
 export const EmployerSiteBatchSchema = yup.array().of(
   yup.lazy((item, options) => {
```

Comment:
This doesn't appear to be used anywhere - Didn't want to remove as part of this PR, but wanted to raise it as tech debt.